src/app/app.html

```diff
@@ -1 +1,19 @@
-<ion-nav [root]="rootPage"></ion-nav>
+<ion-menu side="right" [content]="content">
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>Menu</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-list>
+      <button menuClose ion-item *ngFor="let p of pages" 
```

Comment:
@tokland We would need some logic here. As Wifi Calling and Admin can not be enabled at the same time. I have created a [card for this](https://github.com/EyeSeeTea/FIRE-WiFiCalling/issues/14)